x/incentives/keeper/distribute.go

```diff
@@ -222,9 +230,10 @@ func (k Keeper) doDistributionSends(ctx sdk.Context, distrs *distributionInfo) e
 }
 
 // distributeSyntheticInternal runs the distribution logic for a synthetic rewards distribution gauge, and adds the sends to
-// the distrInfo computed. It also updates the gauge for the distribution.
+// the distrInfo struct. It also updates the gauge for the distribution.
 // locks is expected to be the correct set of lock recipients for this gauge.
 // TODO: Make this code have way more re-use with distribute internal (post-v7).
+// TODO: Create issue for the above
 func (k Keeper) distributeSyntheticInternal(
 	ctx sdk.Context, gauge types.Gauge, locks []lockuptypes.PeriodLock, distrInfo *distributionInfo,
 ) (sdk.Coins, error) {
```

Comment:
Make an issue to have distributeSyntheticInternal re-use distributeInternal logic

Reply:
https://github.com/osmosis-labs/osmosis/issues/2007